### PR TITLE
PP-1029: Travis needs to be updated to install man command

### PR DESCRIPTION
**Pull request by hirenvadalia**
#### Issue-ID

* **[PP-1029](https://pbspro.atlassian.net/browse/PP-1029)**

#### Problem description
* No man command found

#### Cause / Analysis
* man package is not installed in docker containers

#### Solution description
* Install man-db (or man) package in docker containers

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [ ] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [x] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [ ] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by hirenvadalia** [comment]
Since this is Travis CI changes not new PTL tests or manual test require!

File: .travis/opensuse.sh [hunk 1]
```diff
@@ -16,3 +16,5 @@ ${DOCKER_EXEC} rpmbuild -bb pbspro.spec
 ${DOCKER_EXEC} /bin/sh -c 'zypper --no-gpg-checks -n install /root/rpmbuild/RPMS/x86_64/pbspro-server-*.x86_64.rpm'
 ${DOCKER_EXEC} /etc/init.d/pbs start
 ${DOCKER_EXEC} zypper -n install python-pip sudo which net-tools
+${DOCKER_EXEC} zypper -n install man
```
Comment (minghui-liu) [inline .travis/opensuse.sh]: Please combine this line with the previous line.

File: .travis/centos_7.sh [hunk 2]
```diff
@@ -15,4 +15,5 @@ ${DOCKER_EXEC} rpmbuild -bb pbspro.spec
 ${DOCKER_EXEC} /bin/sh -c 'yum -y install /root/rpmbuild/RPMS/x86_64/pbspro-server-*.x86_64.rpm'
 ${DOCKER_EXEC} /etc/init.d/pbs start
 ${DOCKER_EXEC} yum -y install python-pip sudo which net-tools
+${DOCKER_EXEC} yum -y install man-db
```
Comment (minghui-liu) [inline .travis/centos_7.sh]: Please combine this line with the previous line.

**Comment by hirenvadalia** [comment]
@minghui-liu Thanks for review... I have addressed your comments, please review them

**Comment by hirenvadalia** [comment]
@arungrover I believe one of test cases need it... see https://github.com/PBSPro/pbspro/pull/80 for more details...

**Comment by hirenvadalia** [comment]
@minghui-liu and @arungrover Thanks for reviews... I have rebased and squashed commits and now its ready for merge...

**Comment by arungrover** [comment]
Well, that test case does not run during travis build anymore but I see no harm in adding time package to the machines. merging it in

**Pull request merged**